clang/test/Driver/config-file3.c

```diff
@@ -226,3 +226,26 @@
 //
 // RUN: HOME=%S/Inputs/config %clang -### --config-user-dir=~ -v 2>&1 | FileCheck %s --check-prefix=CHECK-TILDE
 // CHECK-TILDE: User configuration file directory: {{.*}}/Inputs/config
+
+//--- Fallback to stripping OS versions
+//
+// RUN: touch %t/testdmode/x86_64-apple-darwin23.6.0-clang.cfg
+// RUN: touch %t/testdmode/x86_64-apple-darwin23-clang.cfg
+// RUN: touch %t/testdmode/x86_64-apple-darwin-clang.cfg
+// RUN: %clang -target x86_64-apple-darwin23.6.0 --config-system-dir=%t/testdmode --config-user-dir= -no-canonical-prefixes --version 2>&1 | FileCheck %s -check-prefix DARWIN --implicit-check-not 'Configuration file:'
```

Comment:
Prefer `--target=`. `-target ` has been deprecated for a long time.

No need to have `^//$` lines.

Reply:
It actually spells `-target` for Darwin.

Reply:
> No need to have `^//$` lines.

The whole file already has it, so this is just matching the style. Is the preferred approach to match the file's style or to use the newer style for new tests (and maybe go back an update old tests in a cleanup PR at some point)?